Extensions/people_notifier.js

```diff
@@ -1,8 +1,8 @@
 //* TITLE Blog Tracker **//
-//* VERSION 0.4.1 **//
+//* VERSION 0.5.0 **//
 //* DESCRIPTION Track people like tags **//
 //* DEVELOPER STUDIOXENIX **//
-//* DETAILS Blog Tracker lets you track blogs like you can track tags. Add them on your dashboard, and it will let you know how many new posts they've made the last time you've checked their blogs, or if they've changed their URLs. **//
+//* DETAILS Blog Tracker lets you track blogs like you can track tags. Add them on your dashboard, and it will let you know how many new posts they've made the last time you've checked their blogs, or if they've changed their URLs.<br><br>Please be aware that the more blogs you add, the longer it will take to track them all. **//
```

Comment:
Do we want to bold or emphasize this as a warning somehow? (Also maybe something like "Please be aware that adding more than ten blogs may cause the tracker extension to load slowly" so they have a sense of how far they can go before they have to start being concerned?)